app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+  after_save :update_user_posts_count
+
+  def update_user_posts_count
+    author.update_posts_count
+  end
+
+  def update_likes_count
+    update(likesCounter: likes.count)
+  end
+
+  def update_comments_count
+    update(commentsCounter: comments.count)
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 `Post` model should has also a method that returns the `5` most recent comments for a given post 😃 
- [ ] `[HINT]`
```
# app/models/post.rb
...

def recent_comments
  comments.order('created_at Desc').limit(5)
end

...
```